contracts/0.4.24/Lido.sol

```diff
@@ -642,7 +642,6 @@ contract Lido is Versioned, StETHPermit, AragonApp {
         require(canDeposit(), "CAN_NOT_DEPOSIT");
 
         IWithdrawalQueue withdrawalQueue = IWithdrawalQueue(locator.withdrawalQueue());
-        require(!withdrawalQueue.isBunkerModeActive(), "CANT_DEPOSIT_IN_BUNKER_MODE");
```

Comment:
Why so?

Reply:
it's design decision based on the current bunker mode's derivation